src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import circleShadow from './CIRCLE_BUTTON_SHADOW.svg';
+import linkedinIcon from './LINKEDINLOGO.svg';
+import githubIcon from './GITHUBLOGO.svg';
+
+class Footer extends Component {
+  render() {
+    return (
+      <footer className="wrapper pt-5 pb-5 flex justify-between items-center">
+        <div className="flex">
+          <h4>Open Library Project</h4>
```

Comment:
It can be `<h2>` - it seems a first heading  of this section - footer.

Reply:
Can I use just <p>?